tests/ContainerTest.php

```diff
@@ -435,6 +592,54 @@ public function testContainerResolvesRegisteredCallable()
         $this->assertInstanceOf('League\Container\Test\Asset\BazInterface', $foo->bar->baz);
     }
 
+    /**
+     * Assert container resolves a registered named function via call
+     *
+     * @return void
+     */
+    public function testContainerResolvesRegisteredNamedFunction()
+    {
+        $c = new Container;
+
+        $c->invokable('function', '\League\Container\Test\Asset\withArgument')->withArgument('hello');
+
+        $this->assertSame($c->call('function'), 'hello');
+    }
+
+    /**
+     * Asserts that a class method is registered as a callable when aliased as string
+     *
+     * @return void
+     */
+    public function testContainerResolvesRegisteredClassMethodWhenRegisteredAsString()
+    {
+        $c = new Container;
+
+        $c->invokable('League\Container\Test\Asset\BazStatic::qux');
+
+        $this->assertSame($c->call('League\Container\Test\Asset\BazStatic::qux'), 'qux');
+    }
+
+    /**
+     * Asserts that an exception is thrown when attempting to register a non
+     * callable
+     *
+     * @return void
+     */
+    public function testExceptionThrownWhenRegisteringNonCallable()
+    {
+        $this->setExpectedException('InvalidArgumentException');
+
+        $c = new Container;
```

Comment:
I think I saw this one first, then scrolled around. If I made that `(new Container)->` here I would have been right. :)